pterodactyl/remote/sftp.md

```diff
@@ -0,0 +1,41 @@
+### `POST /api/remote/sftp/auth`
+
+Authenticate a set of credentials and return the associated server details for a SFTP connection on the daemon. This supports both public key and password based credentials.
+
+### Body
+
+| Name           | Visibility | Type   | Description                                                                                       |
+| -------------- | ---------- | ------ | ------------------------------------------------------------------------------------------------- |
+| type           | required   | string | Determine if the password should check authentication with either the "password" or "public_key". |
+| username       | required   | string | The username of the user attempting to login.                                                     |
+| password       | required   | string | The password of the user attempting to login.                                                     |
+| ip             | optional   | string | The remote address of the connection.                                                             |
+| session_id     | optional   | []byte | The connection's session id.                                                                      |
+| client_version | optional   | []byte | The connection's client version.                                                                  |
```

Comment:
These are not valid fields, see [SftpAuthenticationFormRequest.php](https://github.com/pterodactyl/panel/blob/b23f3114e4cb71406e6de17d0dca0d88205f56c1/app/Http/Requests/Api/Remote/SftpAuthenticationFormRequest.php).

Reply:
oh sorry. I was taking the types from wings, and sometimes there are unnecessary request bodies. will fix later.